lego/apps/polls/models.py

```diff
@@ -0,0 +1,69 @@
+from django.conf import settings
+from django.db import models, transaction
+from django.db.models import Sum
+from django.utils import timezone
+from rest_framework.exceptions import ParseError, PermissionDenied
+
+from lego.apps.content.models import Content
+from lego.apps.polls.permissions import PollPermissionHandler
+from lego.apps.users.models import User
+from lego.utils.models import BasisModel
+
+
+def get_time_delta():
+    return timezone.now() + timezone.timedelta(weeks=52)
+
+
+class Poll(Content, BasisModel):
+
+    description = models.TextField(blank=True)
+
+    valid_until = models.DateTimeField(default=get_time_delta)
+
+    answered_users = models.ManyToManyField(User, related_name="answered_polls")
```

Comment:
As long as voting is done in a trasaction, that should be no problem. The biggest reason to keep the current design is privacy, and imo. that wins. We should also do the same for surveys.

If we need more "data" on each vote (eg. grade from user), we should add that to some kind of separate table; to keep the privacy at a ok level.

Reply:
I don't think a database transaction helps here, it will only ensure that operations are ACID.  
Since it is a poll I don't think anybody expects it to be anonymous, it's also quite common to be able to see what you answered in the result box. But I guess it's nice to have it anonymous even if you lose that one UI feature.

Reply:
Ah okay I guess the many-to-many relation will crash if you try to insert a duplicate, in that case the transaction will be enough

Reply:
Ahh. Haven't thought about the fact that users may want to see what they voted for tho.. :thinking: That might bee smooth

Reply:
bee :bee: